pop-api/src/v0/nonfungibles/types.rs

```diff
@@ -178,18 +180,30 @@ mod tests {
 	#[test]
 	fn ensure_destroy_witness() {
 		assert_eq!(
-			DestroyWitness { item_metadatas: 0, item_configs: 0, attributes: 0 }.encode(),
-			pallet_nfts::DestroyWitness { item_metadatas: 0, item_configs: 0, attributes: 0 }
-				.encode()
+			DestroyWitness {
+				item_metadatas: u32::MAX,
+				item_configs: u32::MAX,
+				attributes: u32::MAX
+			}
+			.encode(),
+			pallet_nfts::DestroyWitness {
+				item_metadatas: u32::MAX,
+				item_configs: u32::MAX,
+				attributes: u32::MAX
+			}
+			.encode()
 		);
 	}
 
 	#[test]
 	fn ensure_mint_witness() {
 		assert_eq!(
-			MintWitness { owned_item: None, mint_price: None }.encode(),
-			pallet_nfts::MintWitness::<ItemId, Balance> { owned_item: None, mint_price: None }
-				.encode()
+			MintWitness { owned_item: Some(ItemId::MAX), mint_price: Some(Balance::MAX) }.encode(),
+			pallet_nfts::MintWitness::<ItemId, Balance> {
+				owned_item: Some(ItemId::MAX),
+				mint_price: Some(Balance::MAX)
```

Comment:
Probably wise to always use a static type as the check value, that way any type aliases changing will be caught. A parachain bricked once due to a change in the nonce type which went undetected.

In other words, we use `u128::MAX| instead so it's explicit.